README.md

```diff
@@ -12,3 +12,19 @@ A super minimal dashboard which lists each available site for all parked paths a
 E.g. `git clone https://github.com/aaemnnosttv/valet-dashboard.git dashboard`
 1. Move the new directory into a Valet-parked path or run `valet link` within it
 1. Run `valet open` from within the directory!
+
+## Setup as default site
+If you would like to access your dashboard by going to http://localhost,
+you have to adjust the Valet config under `~/config/valet/config.json`.
+
+Add the "default" key, like this:
+```json
+{
+    "domain": "dev",
+    "paths": [
+        "/Users/yourname/yourpath"
+    ],
+    "default": "/Users/yourname/yourpath/valet-dashboard",
```

Comment:
Let's make this something a bit more generic since it could technically go anywhere. Something like `/path/to/valet-dashboard/` should be good 👍